glue/main.py

```diff
@@ -40,11 +40,6 @@ def load_plugins(splash=None, require_qt_plugins=False, plugins_to_load=None):
     #
     # where ``setup`` is a function that does whatever is needed to set up the
     # plugin, such as add items to various registries.
-
-    import setuptools
-    logger.info("Loading external plugins using "
-                "setuptools=={0}".format(setuptools.__version__))
```

Comment:
I'd like to keep 'Loading external plugins' - just don't give the setuptools version